Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -567,7 +567,7 @@
             "version": "KqlParameterItem/1.0",
             "name": "SID",
             "type": 2,
-            "query": "SapNetweaver_GetSystemInstanceList_CL\r\n| distinct SID_s",
+            "query": "SapNetweaver_SMON_CL\r\n| where isnotempty(SID_s)\r\n| distinct SID_s",
```

Comment:
@hsridharan, why is the table name changed to SapNetweaver_SMON_CL from SapNetweaver_GetSystemInstanceList_CL?

Reply:
@hsridharan  please fix this

Reply:
@kavitha2270  - Thanks for catching it. Copy paste error